src/main/scala/com/horizon/exchangeapi/tables/Patterns.scala

```diff
@@ -18,19 +18,22 @@ final case class PDataVerification(enabled: Boolean, URL: String, user: String,
 final case class OneUserInputService(serviceOrgid: String, serviceUrl: String, serviceArch: Option[String], serviceVersionRange: Option[String], inputs: List[OneUserInputValue])
 final case class OneUserInputValue(name: String, value: Any)
 
+final case class OneSecretBindingService(serviceOrgid: String, serviceUrl: String, serviceArch: Option[String], serviceVersionRange: Option[String], secrets: List[OneSecretBindingValue])
+final case class OneSecretBindingValue(vaultSecret: String)
```

Comment:
Structurally `"secretBinding": [...]` is almost exactly the same as `"userInput": [...]`. Make sure the plumbing, datatypes, and Swagger examples match structurally. Under `"secrets": [...]` you are looking to store a `List[Object[String, String]]` generically speaking (Look to `OneUserInputValue` to see what is missing).